src/Sylius/Component/Core/Dashboard/DashboardStatisticsProvider.php

```diff
@@ -33,7 +33,7 @@ public function __construct(
         $this->customerRepository = $customerRepository;
     }
 
-    public function getStatisticsForChannel(ChannelInterface $channel): DashboardStatistics
+    public function getStatisticsForChannel(ChannelInterface $channel): DashboardStatisticsInterface
```

Comment:
Also, why didn't you adjust the second method from this class as well?

Reply:
It's totally an oversight, sorry.